docs/personalization/api_reference/tracking_api.md

```diff
@@ -369,24 +379,23 @@ There are no query string parameters for this event. 
 
 ## Tracking events based on recommendations
 
-Tracking events based on integrated recommendations are the only way to measure 
-success of recommendations. 
-It is crucial to inform the Personalization server about which recommendations 
-were shown and what recommendations were clicked. 
-Otherwise, reliable statistics cannot be calculated and used to check against 
-the customer's KPIs.
-
-A recommendation response already includes the requests to generate a Clickrecommended 
-or Rendered event. 
-They are used and executed when a recommendation is clicked/accepted or a recommendation 
-is shown. 
+Tracking events based on integrated recommendations are the only way to measure the accuracy 
+and effectiveness of recommendations. 
+Both recommendation response and trigger message include requests to generate these events.
+Events of this type inform the Personalization server which recommendations 
+were shown to the user and which of those recommendations were clicked. 
+Otherwise, it would be impossible to calculate reliable statistics that could be checked 
+against the customer's KPIs.
+
+A recommendation response includes requests to generate a Rendered and Clickrecommended event. 
+They are executed when a recommendation is shown to the user and when it is clicked or otherwise accepted. 
 Sending Rendered events causes as many requests as recommendations to be displayed, 
 a Clickrecommended event is usually sent only once (when a user clicks on 
 a specific recommendation item).
 
 Example of a recommendation response:
 
-``` jsonp
+``` json
```

Comment:
I think it would be better to remove comments from the code and add it into the table with descriptions.